src/pages/api/question/create.ts

```diff
@@ -0,0 +1,60 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import firestore from "../../../services/firebase/firestore";
+
+import * as yup from "yup";
+import { Question, QuestionCategory } from "../../../types/question";
+
+const schema = yup.object().shape({
+  assignee: yup.object().shape({
+    name: yup.string().required(),
+  }),
+  title: yup.string().required(),
+  description: yup.string().required(),
+  category: yup.string().oneOf(Object.values(QuestionCategory)).required(),
+  complexity: yup.number().min(0).max(5).required(),
+});
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  try {
+    if (req.method === "POST") {
+      const { body } = req;
+
+      if (!(await schema.isValid(body))) {
+        return res.status(404).end("Data format invalid");
```

Comment:
Para payload inválido utiliza-se o status code `400`.
404 para not found (rotas, usuarios)